src/cmake/compiler.cmake

```diff
@@ -580,7 +579,11 @@ if (LINKSTATIC)
     if (WIN32)
         set (CMAKE_FIND_LIBRARY_SUFFIXES .lib .a ${CMAKE_FIND_LIBRARY_SUFFIXES})
     else ()
-        set (CMAKE_FIND_LIBRARY_SUFFIXES .a ${CMAKE_FIND_LIBRARY_SUFFIXES})
+        set (CMAKE_FIND_LIBRARY_SUFFIXES .a)
+    endif ()
+
+    if (MSVC)
+        add_compile_options (/MT)
```

Comment:
Shouldn't be needed if we don't build for Python 2.7.

Reply:
Removed